cypress/support/fragments/marcAuthority/marcAuthoritiesDelete.js

```diff
@@ -0,0 +1,31 @@
+import { Button, PaneHeader, DropdownMenu, Heading, Modal, HTML } from '../../../../interactors';
+
+const actionsButton = PaneHeader({ id: 'paneHeadermarc-view-pane' }).find(Button('Actions'));
+const deleteButton = DropdownMenu().find(Button('Delete'));
+const deleteConfirmModal = Modal({ id: 'confirm-delete-note' });
+const confirmDeleteButton = deleteConfirmModal.find(Button('Delete'));
+
+export default {
+  clickDeleteButton() {
+    cy.do([
+      actionsButton.click(),
+      deleteButton.click(),
+    ]);
+  },
+  checkDeleteModal() {
+    cy.expect([
+      Heading({ id: 'confirm-delete-note-label' }).exists(),
+      deleteConfirmModal.find(Button('Cancel')).exists(),
+      confirmDeleteButton.exists(),
+    ]);
+  },
+  confirmDelete() {
+    cy.do(confirmDeleteButton.click());
+  },
+  checkDelete(headingReference) {
+    cy.expect(HTML(`MARC authority record ${headingReference} has been deleted`).exists());
```

Comment:
try to use HTML interactor with parent interactor.
for example better to use and for debaggin in future:

cy.expect(Pane().find(HTML(`MARC authority record ${headingReference} has been deleted`)).exists());

u can use necessary interactor instead of Pane. it's just example

Reply:
done